third_party/hostport-allocator/pkg/config/kubernetes.go

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
For files in `third_party`, you should not modify copyright header, because you don't "own" them.

The only requirement is to include their license file at the root of the library, like `third_party/hostport-allocator/LICENSE`. The rest of code should be unmodified from its original version. (Or you need to briefly explain what has been changed from the original version.)